src/style/mixins.scss

```diff
@@ -602,3 +602,19 @@ $narrow-dialog-breakpoint: 760px;
         @content;
     }
 }
+
+@mixin table-row-indicator($border-color, $z-index) {
+    &:before {
+        $width-of-selected-row-indicator: 0.2rem;
+        content: '';
+        display: inline-block;
+        box-sizing: border-box;
+        position: sticky;
+        //z-index: $table--has-interactive-rows--selectable-row--hover + 1;
+        z-index: $z-index;
+        inset: 0 0 auto 0;
+        border: $width-of-selected-row-indicator solid $border-color;
+        border-radius: 1rem;
+        margin-right: -$width-of-selected-row-indicator * 2;
+    }
+}
```

Comment:
this is great! However you can keep this mixin in the same file where it is used. This is because it is not a reusable mixin for any other component. We only use this file (`mixins.scss`) for shared mixins (both in components in this codebase, and in other consumers of Lime Elements). Local mixins are better to be kept where they are used.